solution/backend/content_search/migrations/0003_contentindex_uid.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 3.2.22 on 2023-10-24 09:48
+
+from django.db import migrations, models
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('content_search', '0002_add_vector'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='contentindex',
```

Comment:
UID will add a level of protection from people just going through our index 1,2,3 and updating content.  Index changes whenever it is updated so its a pretty safe way of doing it.